File: android/src/main/java/com/scandit/capacitor/datacapture/id_europe_driving_license/ScanditIdEuropeDrivingLicenseNativePlugin.java
package com.scandit.capacitor.datacapture.id_europe_driving_license;

import com.getcapacitor.Plugin;
import com.getcapacitor.PluginCall;
import com.getcapacitor.PluginMethod;
import com.getcapacitor.annotation.CapacitorPlugin;

@CapacitorPlugin(name = "ScanditIdEuropeDrivingLicenseNative")
public class ScanditIdEuropeDrivingLicenseNativePlugin extends Plugin {
  @PluginMethod
  public void echo(PluginCall call) {
    call.resolve();
  }
}

File: ios/Sources/ScanditIdEuropeDrivingLicenseNativePlugin/ScanditIdEuropeDrivingLicenseNativePlugin.swift
import Capacitor
import Foundation

/// Please read the Capacitor iOS Plugin Development Guide
/// here: https://capacitorjs.com/docs/plugins/ios
@objc(ScanditIdEuropeDrivingLicenseNativePlugin)
public class ScanditIdEuropeDrivingLicenseNativePlugin: CAPPlugin, CAPBridgedPlugin {
    public let identifier = "ScanditIdEuropeDrivingLicenseNativePlugin"
    public let jsName = "ScanditIdEuropeDrivingLicenseNative"
    public let pluginMethods: [CAPPluginMethod] = [
        CAPPluginMethod(name: "echo", returnType: CAPPluginReturnPromise)
    ]
    @objc func echo(_ call: CAPPluginCall) {
        call.resolve()
    }
}
